modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/Xref_update_metazoa_conf.pm

```diff
@@ -0,0 +1,64 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2017] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::Xref_update_metazoa_conf;
```

Comment:
Wouldn't this be more appropriately named Xref_update_wormbase_conf?

Reply:
I have no need for this file, and will not need a config checked into the repository as long as there's a good Ensembl config there that I can filter and extend with a small program I wrote.